WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayInformationViewModel.swift

```diff
@@ -2,16 +2,19 @@ import Foundation
 import Combine
 import Yosemite
 
-final class SetUpTapToPayInformationViewModel: PaymentSettingsFlowPresentedViewModel {
+final class SetUpTapToPayInformationViewModel: PaymentSettingsFlowPresentedViewModel, ObservableObject {
     private(set) var shouldShow: CardReaderSettingsTriState = .isUnknown
     var didChangeShouldShow: ((CardReaderSettingsTriState) -> Void)?
     var didUpdate: (() -> Void)?
     let learnMoreURL: URL
     private let stores: StoresManager
 
+    @Published var enableSetup: Bool = true
```

Comment:
Not a blocker: Should we limit access control for this variable?

```suggestion
    @Published private(set) var enableSetup: Bool = true
```

Reply:
Good spot. I'll do that in a following PR as I've already based another one on this.